pkg/project/testdata/svn/format

```diff
@@ -0,0 +1 @@
+12
```

Comment:
The svn binary depends on all these files?

Reply:
Thanks for asking it. I'd never tried removing files from `SVN` but removed some of them and the tests are still passing.